charts/stable/deepstack/values.yaml

```
@@ -54,3 +54,28 @@ persistence:
   modelstore:
     enabled: true
     mountPath: "/modelstore/detection"
+
+configmap:
+  deepstack:
+    enabled: true
+    data:
+      {{- if eq .Values.deepstack.vision_face true }}
+      VISION-FACE: "True"
+      {{- else }}
+      VISION-FACE: "False"
+      {{- end }}
```

Afaik correct synax would be:
```suggestion
      VISION-FACE: {{ ternary "true" "false" .Values.deepstack.vision_face}}
```

---

That helped a lot :)